package.json

```diff
@@ -32,7 +32,7 @@
     "grunt-simple-mocha": "^0.4.0",
     "testee": "0.1.1",
     "jquery": ">2.0",
-    "babel-core": "~4.1.1"
+    "babel-core": "~5.8.14"
```

Comment:
can we pick this to 5.8.14 (remove the ~).  The reason is otherwise it is easy to accidentally check in babel-core when you run "grunt build" for unrelated changes.